calculator.py

```diff
@@ -0,0 +1,11 @@
+user_input = input("Type in your yearly salary = ")
+
+salary = float(user_input)
+
+tax = 0.18
+
+income = salary - (salary * tax)
+
+print("This is your remaining yearly gross income:")

```

Comment:
Use a conditional here, is salary is below something then they have majored in economics.

Reply:
done